### PR TITLE
Fix front page on smaller screens

### PR DESCRIPTION
**Pull request by emitche**
- Move media queries into page where queries apply
- Clean up media queries
- Fix navigation across pages

### CODE REVIEW

File: app/assets/stylesheets/media.css.scss [hunk 1]
```diff
@@ -135,47 +36,9 @@
...
-  }
-
-  .about .main {
-    .aboutMain {
-      width: 100%;
-    }
-  }
-
-}
+}
```
Comment (jpdimond) [inline app/assets/stylesheets/media.css.scss]: new line at end of file

**Comment by jpdimond** [comment]
Looks great! Just one nit.

**Comment by emitche** [comment]
Make sure to add a new line?

**Comment by emitche** [comment]
Merged.

**Pull request closed without merging**